pyproject.toml

```diff
@@ -2,10 +2,8 @@
 name = "cumulus-library"
 requires-python = ">= 3.11"
 dependencies = [
-    "ctakesclient >= 1.3",
     "cumulus-fhir-support >= 1.2",
     "duckdb >= 1.1",
-    "fhirclient >= 4.1",
```

Comment:
Unrelated to rest of PR, I just noticed this.

`ctakesclient` I don't think was ever used by Library - might have been an ancient copy/paste issue from ETL.

`fhirclient` isn't used directly, only indirectly via `cumulus-fhir-support`, so let's let it manage its own versioning needs.